src/app/app.module.ts

```diff
@@ -37,14 +38,16 @@ import { AuthenticationPageComponent } from './authentication-pages/components/a
 import { AboutComponent } from './about/about.component';
 import { OpenCaseListSliderModule } from './shared/open-case-list-slider/open-case-list-slider.module';
 import { UserExperienceBubbleListModule } from './shared/user-experience-bubble-list/user-experience-bubble-list.module';
+import { FaqPageComponent } from './faq-page/faq-page.component';
+import { FaqBlockComponent } from './faq-block/faq-block.component';
 
 // eslint-disable-next-line prefer-arrow/prefer-arrow-functions
 export function createTranslateLoader(http: HttpClient) {
   return new TranslateHttpLoader(http, './assets/i18n/', '.json');
 }
 
 @NgModule({
-  declarations: [AppComponent, MainLayoutComponent, FooterOnlyLayoutComponent, AboutComponent],
+  declarations: [AppComponent, MainLayoutComponent, FooterOnlyLayoutComponent, AboutComponent, FaqPageComponent, FaqBlockComponent],
```

Comment:
To stick with the rest of the code you could put these two components into a separate `FaqModule`.
That way everything regarding FAQ is encapsulated into it's own module and the main `AppModule` only imports all the "feature modules"

Reply:
Done